Recording/Stages/SendDreamlandInfo.asm

```diff
@@ -0,0 +1,59 @@
+################################################################################
+# Address: 80211bf8
+################################################################################
+.include "Common/Common.s"
+.include "Recording/Recording.s"
+
+################################################################################
+# Routine: SendDreamlandInfo
+# ------------------------------------------------------------------------------
+# Sends whispy wind direction when it changes.
+################################################################################
+
+.set REG_LR,6
+.set REG_Buffer,7
+.set REG_BufferOffset,8
+
+b Start
+STATIC_PREVIOUS_VALUE:
+  blrl
+  .long 0x00000000
+
+Start:
+  mtlr REG_LR
```

Comment:
Hmm just realized I meant this to be mflr. However, now I'm wondering if it's necessary at all to backup the lr because the gecko code worked anyways...